### PR TITLE
Pagination & filters 

### PR DESCRIPTION
**Pull request by joelmayemba**
Pagination system & filters corrected in JS + CSS for the buttons

### CODE REVIEW

**Review by chouaibj — APPROVED** [review]

File: src/View/Mocktails/index.html.twig [hunk 1]
```diff
@@ -1,71 +1,65 @@
...
+            </ul>
+            <div class="row mocktails-box">
+                {% for mocktail in mocktails %}
+                    <div class="col-lg-3 col-md-6 col-sm-12 align-self-center mb-30 properties-items col-md-6" data-tag="{{ mocktail['CATEGORY_ID'] }}">
+                        <div class="item text-center" data-product-name="{{ mocktail.NAME }}">
+                            <a href="mocktaildetail?product={{ mocktail.ID | url_encode }}">
+                                <img src="{{ 'assets/images/mocktails/' ~ mocktail.IMG }}" alt="{{ mocktail.NAME }}"/>
+                            </a>
+                            <span class="category">{{ mocktail.CATEGORY_NAME }}</span>
+                            <h4>{{ mocktail.NAME }}</h4>
```
Comment (Benja-san) [inline src/View/Mocktails/index.html.twig:39]: if you have h4 you should have h1 - h2 - h3 .

h1 should be mandatory on a referenced page

File: src/View/Mocktails/show.html.twig [hunk 2]
```diff
@@ -15,47 +16,68 @@
...
 {% block content %}
 	<div class="section title-detail-page">
 		<div class="container">
 			<div class="row">
 				<div class="col-lg-4">
 					<div class="section-heading">
-						<a href="/mocktails">| Retour à la page Nos Mocktails</a>
-						<h2>{{ mocktail.NAME }}</h2>
+						<a href="/mocktails"><i class="fa-solid fa-arrow-left"></i></a>/{{ mocktail.NAME }}
+						{# <h2>{{ mocktail.NAME }}</h2> #}
```
Comment (Benja-san) [inline src/View/Mocktails/show.html.twig:27]: dead code

File: src/View/Mocktails/show.html.twig
Comment (Benja-san) [inline src/View/Mocktails/show.html.twig:1]: no H1 on this page neither

File: src/View/Mocktails/show.html.twig [hunk 3]
```diff
@@ -64,5 +86,6 @@
 			</div>
 		</div>
 	</div>
-</div>
+	{# {{ dump(mocktail)}}; #}
```
Comment (Benja-san) [inline src/View/Mocktails/show.html.twig:89]: remove dump

**Review by Benja-san — CHANGES_REQUESTED** [review]

File: src/View/Mocktails/show.html.twig
Comment (Benja-san) [inline src/View/Mocktails/show.html.twig:1]: change h3 into H1

**Review by Benja-san — CHANGES_REQUESTED** [review]

File: src/View/Mocktails/show.html.twig
Comment (joelmayemba) [reply src/View/Mocktails/show.html.twig:1]: done

**Review by Benja-san — APPROVED** [review]

**Pull request merged**